systers_portal/systers_portal/settings/base.py

```diff
@@ -164,3 +164,14 @@
 
 # Django Crispy Forms configuration
 CRISPY_TEMPLATE_PACK = 'bootstrap3'
+
+# Settings for Facebook OAuth
+SOCIALACCOUNT_PROVIDERS = \
+    {'facebook': {'METHOD': 'oauth2',
+                  'SCOPE': ['email'],
+                  'AUTH_PARAMS': {'auth_type': 'reauthenticate'},
+                  'FIELDS': ['id', 'email'],
+                  'EXCHANGE_TOKEN': True,
+                  'LOCALE_FUNC': lambda request: 'kr_KR',
+                  'VERIFIED_EMAIL': False,
+                  'VERSION': 'v2.4'}}
```

Comment:
The default version is `7.0` now, any reasons for using such an old version? Again, please give thought to each of these.